docs/user-guide/plotting.ipynb

```diff
@@ -5,7 +5,7 @@
    "metadata": {},
    "source": [
     "# Plotting\n",
-    "`scipp` offers a number of different ways to plot data contained in a `Dataset`."
+    "`scipp` offers a number of different ways to plot data from a `DataArray` or a `Dataset`."
```

Comment:
I'm not sure the user needs to know what a `DataArray` is (apart from advanced users). It's just like a `Dataset` but contains only one named field. And by saying "contained in a Dataset" i was kind of covering plotting all the data in a `Dataset` or just a subset of it.

Reply:
I would have agreed until recently about `DataProxy`. But `DataArray` is actually a stand-alone object. `Dataset` is like a `dict`, but `DataArray` is *not* like a dict with one item.